### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by gsaha322** [HITS-MCM/gromacs-ramd#32] [Q&A]
I have GROMACS installed in my machine with proper pre-requisites and it runs properly. Do we need to install anything extra for running RAMD? Please provide a manual and a sample .mdp file for using this RAMD.

**Comment by BerndDoser** [comment]
GROMACS-RAMD is not available in the official GROMACS. Please use the [latest release of GROMACS-RAMD](https://github.com/HITS-MCM/gromacs-ramd/releases/tag/gromacs-2020.5-ramd-2.0) and install it in the same way as the official GROMACS. Alternatively, you can use [EasyBuild](https://github.com/HITS-MCM/gromacs-ramd#hpc-deployment-with-easybuild).
